pkg/resource/resource.go

```diff
@@ -93,6 +99,16 @@ func (r *Resource) AppendRefBy(id resid.ResId) {
 	r.refBy = append(r.refBy, id)
 }
 
+// GetRefVar returns vars that refer to current resource
+func (r *Resource) GetRefVar() []string {
+	return r.refVar
+}
+
+// AppendRefVar appends a var into the refVar list
```

Comment:
`AppendRefVarName(v Var)`

to make this more type safe, demand a Var, but keep only the name.

Reply:
thanks, done